README.md

```diff
@@ -75,6 +64,65 @@ The unit tests have been implemented by sending a POST request to GitHub and obs
 One test case send valid API data and expects a valid response, another sends invalid API data and
 expects an invalid response.
 
+## API Documentation
+
+### POST requests
+`/push-events`
+
+#### Description
+Pushes an event to the server
+
+#### Parameters
+None
+
+Request body implemented according to [this](https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#push)
+
+Request body example:
+```
+{
+    "ref":"refs/heads/mustafa/test-branch",
+    "after":"ea312d0e44d08a603935b8b926eea76e4ed0e266",
+    "repository": {
+        "name":"a2-ci-test",
+        "clone_url":"https://github.com/DD2480-G12/a2-ci-test.git",
+        "owner": {
+            "name":"DD2480-G12"
+        }
+    }
+}
+```
+#### Responses
+204 No Content: This endpoint will always return 204 No Content
```

Comment:
This is not true, it can return other statuses too. What you can say is that it returns 204 if the request is accepted by the CI server